src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,47 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter Interface', () => {
+  let roomsSDKAdapter;
+  let sdk;
+  let user;
+  let createdRoom;
+
+  beforeAll(async () => {
+    // Setting timeout higher for user creation and room creation
+    jest.setTimeout(15000);
+
+    user = await createIntegrationTestUser();
+
+    sdk = createIntegrationSdkInstance(user.token);
+
+    createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+  });
+
+  afterAll(() =>
+    removeTestUser(user).catch((reason) => {
+      // eslint-disable-next-line no-console
+      console.warn('failed to delete test user', reason);
+    })
+  );
+
+  beforeEach(() => {
+    roomsSDKAdapter = new RoomsSDKAdapter(sdk);
+  });
+
+  describe('getRoom() functionality', () => {
```

Comment:
On the component repo, I setup the describe 
```suggestion
  describe('getRoom() returns', () => {
```
[See sample test file](https://github.com/webex/components/blob/master/src/components/WebexActivity/ActivityHeader.test.js#L24) in component repo